save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestSuitesSourceController.kt

```diff
@@ -352,6 +350,35 @@ class TestSuitesSourceController(
         }
         .map { it.toDto() }
 
+    @PostMapping("/api/$v1/test-suites-sources/create")
+    @RequiresAuthorizationSourceHeader
+    @PreAuthorize("permitAll()")
+    @Operation(
+        method = "POST",
+        summary = "Get or create a new test suite source by provided values.",
+        description = "Get or create a new test suite source by provided values.",
+    )
+    @ApiResponse(responseCode = "200", description = "Successfully get or create test suites source with requested values.")
+    @ApiResponse(responseCode = "404", description = "Organization was not found by provided name.")
+    @ApiResponse(responseCode = "404", description = "Git credentials were not found by provided url.")
```

Comment:
Should these be merged? Or does swagger merge them?

Reply:
Oh you are right, they are not merged. Will fix it

Reply:
@nulls used it in the same way, seems like it needs to be changed.

Reply:
Fixed